components/ord-service/src/main/java/com/sap/cloud/cmp/ord/service/storage/model/APIEntity.java

```diff
@@ -119,6 +119,10 @@
     @CollectionTable(name = "ord_labels", joinColumns = @JoinColumn(name = "api_definition_id"))
     private List<Label> labels;
 
+    @ElementCollection
+    @CollectionTable(name = "ord_documentation_labels", joinColumns = @JoinColumn(name = "api_definition_id", referencedColumnName= "id"))
```

Comment:
Why is `referencedColumnName= "id"` needed? In the labels above (which are technically identical) it looks like it is not needed.